frontend/src/Movie/Details/Credits/Cast/MovieCastPoster.js

```diff
@@ -95,23 +96,27 @@ class MovieCastPoster extends Component {
           <div
             style={elementStyle}
           >
-            <MovieHeadshot
-              className={styles.poster}
-              style={elementStyle}
-              images={images}
-              size={250}
-              lazy={false}
-              overflow={true}
-              onError={this.onPosterLoadError}
-              onLoad={this.onPosterLoad}
-            />
-
-            {
-              hasPosterError &&
-                <div className={styles.overlayTitle}>
-                  {personName}
-                </div>
-            }
+            <a href={`https://www.themoviedb.org/person/${tmdbId}`} target="_blank"
+              rel="noopener noreferrer"
+            >
+              <MovieHeadshot
+                className={styles.poster}
+                style={elementStyle}
+                images={images}
+                size={250}
+                lazy={false}
+                overflow={true}
+                onError={this.onPosterLoadError}
+                onLoad={this.onPosterLoad}
+              />
+
+              {
+                hasPosterError &&
+                  <div className={styles.overlayTitle}>
+                    {personName}
+                  </div>
```

Comment:
I considered that but I thought it would be better to do it for the poster as then it does not change the style of the text (or require styling to match current theme). I personally am more likely to click on the picture than the name to go somewhere.

Perhaps the slide itself would be preferred? I just wasn't sure how to implement it for the slide due to being a different set of props being imported (I think?)

Reply:
I don't have the time to implement it in this fashion safely unfortunately. I would rely too heavily on imports which I don't have the knowledge of and I would not be confident that it would not have unexpected consequences if the component was to change.

I had never used those hover links before to realise that was the standard way to implement this kind of feature. It is probably best to close this PR in that case as I won't be able to provide an up-to-par solution for the foreseeable future.